android/app/src/main/AndroidManifest.xml

```diff
@@ -55,6 +56,7 @@
             android:value="2" />
 
         <service android:name="com.ryanheise.audioservice.AudioService"
+            android:foregroundServiceType="mediaPlayback"
```

Comment:
There were other runtime errors that required specifying this

Reply:
We might also need `<uses-permission android:name="android.permission.FOREGROUND_SERVICE_MEDIA_PLAYBACK"/>` in the manifest. (See https://github.com/nt4f04uNd/sweyer/pull/130/files#diff-63272c98a6330850888e633b43ba4c9decee6431a115e0d2731564838eaa1d1dR10)

Reply:
Ah, I see you did that 👍

Reply:
yep, I added it